nautobot/dcim/views.py

```diff
@@ -136,6 +136,11 @@ def post(self, request):
         )
 
 
+class BaseDeviceComponentsBulkRenameView(generic.BulkRenameView):
+    def get_selected_objects_parents_name(self, selected_objects):
+        return selected_objects.first().device.name
```

Comment:
Can we use `.parent.name` so that both VMInterface and all the DeviceComponent views all can share the same code?